Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Carousel/CarouselPage.xaml

```diff
@@ -8,8 +8,9 @@
       mc:Ignorable="d">
 
     <Grid Background="{StaticResource Brush-Grey-05}">
-        <Border Margin="0">
-            <controls:Carousel x:Name="CarouselControl"
+   
+            <Border Margin="0">
```

Comment:
It's also not a blocking review. It's a question wondering why the spacing was added